HARK/core.py

```diff
@@ -939,7 +950,15 @@ def solveAgents(self):
         '''
         #for this_type in self.agents:
         #    this_type.solve()
-        multiThreadCommands(self.agents,['solve()'])
+        try:
+            multiThreadCommands(self.agents,['solve()'])
+        except Exception as err:
+            if self.print_parallel_error_once:
+                # Set flag to False so this is only printed once. 
+                self.print_parallel_error_once = False
+                print("**** WARNING: could not execute multiThreadCommands in HARK.core.Market.solveAgents(), so using the serial version instead. This will likely be slower. The multiTreadCommands() functions failed with the following error:", '\n    ', sys.exc_info()[0], ':', err) #sys.exc_info()[0])
+            multiThreadCommandsFake(self.agents,['solve()'])
```

Comment:
This is a nice solution! Graceful degradation is good.

It might be worth putting it in its own commit.